### PR TITLE
feat(MembershipsAdapter): add addRoomMember method

### PR DESCRIPTION
**Pull request by harsh789rai**
add addRoomMember method to the membershipSDKAdapter.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,13 +30,13 @@
   "author": "devsupport@webex.com",
   "license": "MIT",
   "peerDependencies": {
-    "rxjs": "^6.5.4",
-    "webex": "^1.157.0"
+    "rxjs": "^6.6.7",
+    "webex": "^1.159.4"
```
Comment (kponmuth) [inline package.json:34]: We don't want to update rxjs and webex version Harsh. Is there a need in this PR?

**Review by kponmuth — CHANGES_REQUESTED** [review]

**Comment by kponmuth** [comment]
What about adding membership for a user using his/her email? 

Is that gonna be a separate PR?

File: package.json (hunk 1 above)
Comment (harsh789rai) [reply package.json]: changes done

**Comment by harsh789rai** [comment]
> What about adding membership for a user using his/her email?
> 
> Is that gonna be a separate PR?

Yes there will be a seperate PR

**Review by kponmuth — APPROVED** [review]

**Pull request merged**